### PR TITLE
[TR] PIM-5054 : merge variant group imported template values

### PR DESCRIPTION
**Pull request by jmleroux**
| Q                 | A
| ----------------- | ---
| Specs             |modified
| Behats            |added
| Blue CI           |yes
| Changelog updated |yes
| Review and 2 GTM  |yes

### CODE REVIEW

**Comment by ClementGautier** [comment]
GTM

File: features/import/product_variant_group/import_variant_group_with_template_values.feature [hunk 1]
```diff
@@ -32,3 +32,19 @@ Feature: Execute an import
...
+    And the following job "variant_group_import" configuration:
+      | filePath | %file to import% |
+    When I am on the "variant_group_import" import job page
+    And I launch the import job
+    And I wait for the "variant_group_import" job to finish
+    When I am on the "tshirts" variant group page
+    Then I should see "Description"
+    When I visit the "Media" group
+    Then I should see "Legend"
+
```
Comment (skeleton) [inline features/import/product_variant_group/import_variant_group_with_template_values.feature]: remove extra blank line

**Comment by skeleton** [comment]
baby comment then GTM

File: src/Pim/Bundle/BaseConnectorBundle/Processor/Denormalization/VariantGroupProcessor.php [hunk 2]
```diff
@@ -155,11 +155,15 @@ protected function updateVariantGroupValues(GroupInterface $variantGroup, array
         if (!empty($valuesData)) {
             $values = $this->denormalizeValuesFromItemData($valuesData, $variantGroup->getProductTemplate());
             $this->validateValues($variantGroup, $values, $groupData);
-            $template = $this->getProductTemplate($variantGroup);
+            $template  = $this->getProductTemplate($variantGroup);
```
Comment (phaseinducer) [inline src/Pim/Bundle/BaseConnectorBundle/Processor/Denormalization/VariantGroupProcessor.php]: extra space

**Comment by phaseinducer** [comment]
GTM

**Pull request merged**